### DISCUSSION TITLE
Portrait Pack

### DISCUSSION THREAD

**Discussion by Isewein** [exult/exult#566] [General]
With the Portrait Pack / Keyring mod (really glad it's integrated into the main Exult download now!) bringing SI style portraits to BG, is there any way to add the original SI portraits to the BG selection screen too?

EDIT: I figured out how to do it with this useful tutorial: https://yulotomorrow.wordpress.com/2022/03/18/modding-in-exult-the-ultima-vii-engine-part-1-portrait-editing/

I managed to convert the portraits from Ultima Lazarus too while I was at it - maybe Marzo would like to integrate them into his Portrait Pack?

**Comment by KnightCaptainU7** [comment]
I'd be interested in seeing them. Do you have them posted?
